cl/phase1/network/services/attestation_service.go

```diff
@@ -284,47 +283,47 @@ func (s *attestationService) ProcessMessage(ctx context.Context, subnet *uint64,
 	return ErrIgnore
 }
 
-type attestationJob struct {
-	att          *AttestationWithGossipData
-	creationTime time.Time
-	subnet       uint64
-}
+// type attestationJob struct {
```

Comment:
Can we just remove these unused code?

Reply:
will leave it commented for now, in case we will be needing this again